core/domain/user_jobs_one_off.py

```diff
@@ -1073,3 +1073,52 @@ def reduce(key, values):
 
         if len(values) != 1:
             yield ('FAILURE', values)
+
+
+class OldDraftDeletionJob(jobs.BaseMapReduceOneOffJobManager):
+    """Job that discards any drafts that were last updated in 2019 or prior.
+
+    This is done to avoid issues arising from old schema version
+    incompatibility. It is unlikely that such drafts are being used or relied
+    on anyway, since they have been abandoned for over a year.
+    """
+
+    @classmethod
+    def enqueue(cls, job_id, additional_job_params=None):
+        super(OldDraftDeletionJob, cls).enqueue(job_id, shard_count=64)
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [user_models.ExplorationUserDataModel]
+
+    @staticmethod
+    def map(model):
+        if model.draft_change_list is None:
+            return
+
+        exploration = exp_fetchers.get_exploration_by_id(
+            model.exploration_id, strict=False)
+
+        if exploration is None:
+            yield ('DISCARDED - Exploration is missing', model.id)
+        elif model.last_updated.timetuple().tm_year <= 2019:
```

Comment:
Per comment above, we could repurpose this job to delete all drafts older than XX months, instead, to help keep the datastore clean. That actually could even be a regular cron--it seems reasonable.

Reply:
Same note. I don't think there's a strong need to do this yet, but will consider making it a standard thing if we have to do this again for some reason.